public/js/medid/card.js

```diff
@@ -54,8 +54,8 @@ define(['jspdf', 'jquery', 'medid/hyphenator', 'medid/res'], function(jsPDF, $,
 		var leftStartPos = [12,55], rightStartPos = [38,30];
 		var lineHeight = 5, leftLabelWidth = 19, rightLabelWidth = 21;
 		for (i = 0; i < fields.length && i < creator.cardNum; i++) {
-			fields[i].label = fields[i].label.substring(0, 13);
-			fields[i].field = fields[i].field.substring(0, 19);
+			fields[i].label = fields[i].label.substring(0, 14);
+			fields[i].field = fields[i].field.substring(0, 20);
```

Comment:
When both label and field contain the max amount of characters, they still overlap. I would suggest bringing down the length of the label back to 13, leaving the field length alone, and that should fix it